.github/workflows/vertexai.yml

```diff
@@ -28,6 +28,8 @@ jobs:
       FIREBASECI_USE_LATEST_GOOGLEAPPMEASUREMENT: 1
     steps:
     - uses: actions/checkout@v4
+    - name: Clone mock responses
+      run: scripts/update_vertexai_responses.sh
```

Comment:
Looks like this worked well 🎉 : https://github.com/firebase/firebase-ios-sdk/actions/runs/9669870603/job/26677377013?pr=13186#step:6:393

Would you mind trying to add the exact same step to [`swift-build-run`](https://github.com/firebase/firebase-ios-sdk/blob/901e0cc49da4e25427aea190f9542af0f027b836/.github/workflows/spm.yml#L45) and [`iOS-Device`](https://github.com/firebase/firebase-ios-sdk/blob/901e0cc49da4e25427aea190f9542af0f027b836/.github/workflows/spm.yml#L77) (preferably with a name like "Clone mock Vertex AI responses" since those jobs aren't Vertex-specific)?

Reply:
Thanks for resolving this! [`swift-build-run`](https://github.com/firebase/firebase-ios-sdk/actions/runs/9670491347/job/26679300998?pr=13186) and `iOS-Device` are passing now too.